queries/c/highlights.scm

```diff
@@ -157,11 +157,8 @@
 (comment) @comment
 
 ;; Parameters
-(parameter_declaration
-  declarator: (identifier) @parameter)
-
-(parameter_declaration
-  declarator: (pointer_declarator) @parameter)
+((identifier) @parameter
+  (#has-ancestor? @parameter "parameter_declaration"))
```

Comment:
We probably want to avoid `#has-ancestor` since checking the parent has become expensive.